src/qililab/slurm.py

```diff
@@ -89,7 +89,7 @@ def submit_job(line: str, cell: str, local_ns: dict) -> None:
     # are imported inside the SLURM job)
     notebook_code = "\n".join(local_ns["In"]).split("\n")
     import_lines = [line for line in notebook_code if line.startswith(("import ", "from ")) and "slurm" not in line]
-    executable_code = "\n".join(import_lines + [cell])
+    executable_code = "\n".join([*import_lines, cell])
```

Comment:
It's less legible. Is the old way invalid?

Reply:
This rule is described here: https://docs.astral.sh/ruff/rules/collection-literal-concatenation/

It claims it is more readable and efficient and follows PEP 448: https://peps.python.org/pep-0448/